meta-ibm/recipes-phosphor/webui/webui-vue/ips.app.css

```diff
@@ -0,0 +1 @@
fff!important;background-color:#fff!important}.custom-select[data-v-60dba918]{background:#fff url("data:image/svg+xml;charset=utf-8,%3Csvg xmlns='http://www.w3.org/2000/svg' width='4' height='5'%3E%3Cpath fill='%2336b6ec' d='M2 0L0 2h4zm0 5L0 3h4z'/%3E%3C/svg%3E") right .75rem center/8px 10px no-repeat}.custom-select[data-v-60dba918]:disabled{background:#ccc url("data:image/svg+xml;charset=utf-8,%3Csvg xmlns='http://www.w3.org/2000/svg' width='4' height='5'%3E%3Cpath fill='%23999' d='M2 0L0 2h4zm0 5L0 3h4z'/%3E%3C/svg%3E") right .75rem center/8px 10px no-repeat!important}.custom-control-input:checked~.custom-control-label[data-v-60dba918]:before{border:1px solid #0062fe}.custom-radio .custom-control-input:checked~.custom-control-label[data-v-60dba918]:after{background-image:url("data:image/svg+xml;charset=utf-8,%3Csvg xmlns='http://www.w3.org/2000/svg' width='9' height='9' viewBox='-4 -4 8 8'%3E%3Ccircle r='3'/%3E%3C/svg%3E")}.custom-control-input:disabled~.custom-control-label[data-v-60dba918]:before,.custom-control-input[disabled]~.custom-control-label[data-v-60dba918]:before{border:1px solid #ccc}.custom-control-label[data-v-60dba918]:before{border-color:#36b6ec}.custom-checkbox .custom-control-input:checked~.custom-control-label[data-v-60dba918]:after{background-image:url("data:image/svg+xml;charset=utf-8,%3Csvg xmlns='http://www.w3.org/2000/svg' width='10' height='10' viewBox='0 0 8 8'%3E%3Cpath fill='%23213ebf' d='M6.564.75l-3.59 3.612-1.538-1.55L0 4.26l2.974 2.99L8 2.193z'/%3E%3C/svg%3E")}.custom-checkbox .custom-control-input:checked~.custom-control-label[data-v-60dba918]:after,.custom-checkbox .custom-control-input:indeterminate~.custom-control-label[data-v-60dba918]:before,.custom-control-input:checked~.custom-control-label[data-v-60dba918]:before{background-color:#fff}.custom-switch .custom-control-input:checked~.custom-control-label[data-v-60dba918]:after{background-color:#36b6ec}.app-container[data-v-60dba918] .app-content{background-color:#fafafa!important}.app-container[data-v-60dba918] .nav-container{background-color:#d7f8f3!important}.app-container[data-v-60dba918] .nav-container .btn-link:hover,.app-container[data-v-60dba918] .nav-container .nav-link:hover{background-color:#b7dae9}.app-container[data-v-60dba918] .nav-container .btn-link:active,.app-container[data-v-60dba918] .nav-container .btn-link:focus,.app-container[data-v-60dba918] .nav-container .nav-link:active,.app-container[data-v-60dba918] .nav-container .nav-link:focus,.app-navigation[data-v-60dba918] .nav-link--current{background-color:#36b6ec}.app-navigation[data-v-60dba918] .nav-container .btn-link,.app-navigation[data-v-60dba918] .nav-container .nav-link{color:#3f3f3f!important}.app-container[data-v-60dba918] .nav-link.bg-light,.border-light[data-v-60dba918]{background-color:#eee!important;border-color:#3ac4bf!important}.btn-secondary[data-v-60dba918]{background-color:#eee;border-color:#eee;color:#0f62fe}.btn-dark[data-v-60dba918]{background-color:#fafafa;border-c
```

Comment:
So this is taking the built CSS,
root@rain111bmc:/usr/share/www/css# ls
app.css.gz

and modifying it? 

This would be a lot better if this could go in the *ips branches of webui-vue.. not some post build overwrite

Reply:
https://github.com/ibm-openbmc/webui-vue/pull/51